modules/bmo-troubleshooting-latency-when-deleting-a-baremetalhost-resource.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+// //installing/installing_bare_metal_ipi/installing_bare_metal_ipi/ipi-install-troubleshooting.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="ipi-install-troubleshooothing-latency-when-deleting-a-baremetalhost-resource_{context}"]
+
+= Troubleshooting latency when deleting a BareMetalHost resource
+
+When the Bare Metal Operator (BMO) deletes a `BareMetalHost` resource, Ironic deprovisions the bare-metal host. For example, this might happen when scaling down a machine set. Deprovisioning involves a process known as "cleaning", which performs the following steps:
+
+* Powering off the bare-metal host
+* Booting a service RAM disk on the bare-metal host
+* Removing partitioning metadata from all disks
+* Powering off the bare-metal host again
+
+If cleaning does not succeed, the deletion of the `BareMetalHost` resource will take a long time and might not finish.
```

Comment:
The deletion will proceed after 3 retries of cleaning. I don't quite like this, but that's how it works. The thing is, many customers are too impatient to wait for 3 timeouts.